packages/shared_preferences/shared_preferences_android/android/src/test/java/io/flutter/plugins/sharedpreferences/DeprecatedSharedPreferencesTest.java

```diff
@@ -303,27 +304,27 @@ private static class FakeSharedPreferences implements SharedPreferences {
 
     // All methods below are not implemented.
     @Override
-    public @NonNull boolean contains(@NonNull String key) {
+    public boolean contains(@NonNull String key) {
       throw new UnsupportedOperationException("This method is not implemented for testing");
     }
 
     @Override
-    public @NonNull boolean getBoolean(@NonNull String key, @NonNull boolean defValue) {
+    public boolean getBoolean(@NonNull String key, boolean defValue) {
       throw new UnsupportedOperationException("This method is not implemented for testing");
     }
 
     @Override
-    public @NonNull float getFloat(@NonNull String key, @NonNull float defValue) {
+    public float getFloat(@NonNull String key, float defValue) {
       throw new UnsupportedOperationException("This method is not implemented for testing");
     }
 
     @Override
-    public @NonNull int getInt(@NonNull String key, @NonNull int defValue) {
+    public int getInt(@NonNull String key, int defValue) {
       throw new UnsupportedOperationException("This method is not implemented for testing");
     }
 
     @Override
-    public @NonNull long getLong(@NonNull String key, @NonNull long defValue) {
```

Comment:
Were all of these annotations wrong?

Reply:
android studio was yelling at me about it not being necessary for these primitives

Reply:
Ah, right. I forget when I'm not actively writing Java code that it has primitive and object versions of each numeric type.